include/fmt/ranges.h

```diff
@@ -229,7 +229,7 @@ template <class Tuple, class F> void for_each(Tuple&& tup, F&& f) {
 }
 
 template <typename Range>
-using value_type =
+using uncvref_type =
```

Comment:
I think `value_type` is more descriptive because this gives the range value type. The fact that cvref is removed is less important.

Reply:
It doesn't give the `value_type` though, it gives you the range's `reference` - with cvref stripped. A lot of the time, that _is_ the `value_type`, but not always. For example: for `vector<bool>` the `value_type` is `bool` but this gives you `vector<bool>::reference` (which is what we would actually use to format). Or `zip`ping two `vector<int>`s has a `value_type` of `pair<int, int>` but this gives you `pair<int&, int&>`.

I wanted to change it mainly because calling it `value_type` is misleading - we never use the range's `value_type` for anything.

Reply:
OK but could you clarify this in the comment then?

Reply:
Sure.